TESTING.md

```diff
@@ -0,0 +1,79 @@
+# Testing
+
+We use many different kind of tests to maintain quality in our codebase. Here's an overview of what these tests are, and how to run them.
```

Comment:
`s/kind/kinds`

This document seems to be a summary of our automated testing, so you could use that in the top-level description here.  How about:

> We use automated tests to maintain quality in our codebase.  Here's an overview of the kinds of automated tests we provide, and how to run them.